openapi/multi-yaml/paths/account@{account_id}@payout_holds@{payout_hold_id}.yaml

```diff
@@ -0,0 +1,55 @@
+get:
+  summary: Get a Payout Hold
+  description: Retrieve a specific payout hold by ID.
+  operationId: GetPayoutHold
+  tags:
+    - Payout Holds
+  parameters:
+    - $ref: ../components/parameters/account-id-path.yaml
+    - $ref: ../components/parameters/payout-hold-id-path.yaml
+    - $ref: ../components/parameters/authorization-header.yaml
+  responses:
+    '200':
+      description: Successfully get a payout hold
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: ../components/schemas/Envelope.yaml
+              - properties:
+                  type:
+                    example: payout_hold
+                  data:
+                    $ref: ../components/schemas/PayoutHold.yaml
+
+patch:
+  summary: Update a Payout Hold
+  description: Update an existing payout hold. Currently only supports updating the end_date field.
+  operationId: UpdatePayoutHold
+  tags:
+    - Payout Holds
+  parameters:
+    - $ref: ../components/parameters/account-id-path.yaml
```

Comment:
Same here - see my earlier question about why the account_id is needed in the path.